acceptance/acceptance_test.go

```diff
@@ -462,6 +462,7 @@ func runTest(t *testing.T, dir, coverDir string, repls testdiff.ReplacementsCont
 		require.Len(t, items, 2)
 		cmd.Env = append(cmd.Env, keyvalue)
 		repls.Set(items[1], "["+items[0]+"]")
+		t.Logf("Set env %v", keyvalue)
```

Comment:
We don't need to log for every case, only when we omit from the test name.

Reply:
Fixed